### DISCUSSION TITLE
dmc.Stack breaks dmc.RadarChart 🐛

### DISCUSSION THREAD

**Discussion by pip-install-python** [snehilvj/dash-mantine-components#393] [Q&A] [answered]
Basic example, but if you try and put dmc.RadarChart in a dmc.Stack with align="center", it breaks the dmc.RadarChart. Tested on 0.14.6

align='stretch'
<img width="1398" alt="Screenshot 2024-11-03 at 12 45 30 PM" src="https://github.com/user-attachments/assets/53dc9ed1-3b25-400a-b363-1a67cb8b8357">

align='center'
<img width="1398" alt="Screenshot 2024-11-03 at 12 46 30 PM" src="https://github.com/user-attachments/assets/1265d9c1-2b02-408c-b642-6ee0a0ebcb05">

align="flex-start"
<img width="1432" alt="Screenshot 2024-11-03 at 12 47 26 PM" src="https://github.com/user-attachments/assets/785a50dc-e6b7-482e-9298-4f07f8db8a3f">

align="flex-end"
<img width="1429" alt="Screenshot 2024-11-03 at 12 48 03 PM" src="https://github.com/user-attachments/assets/215d3f0a-0348-4b98-870b-5ea088b7c825">

```
from dash import Dash, _dash_renderer
import dash_mantine_components as dmc


_dash_renderer._set_react_version("18.2.0")

app = Dash(__name__)

radar_data = [
    {"product": "Downloads", "sales_january": 120, "sales_february": 100},
    {"product": "Views", "sales_january": 98, "sales_february": 90},
    {"product": "Rating", "sales_january": 65, "sales_february": 150},
]

app.layout = dmc.MantineProvider(
    [
        dmc.Stack(
            [
                dmc.Text("Most Recent vs Overall:", c="white"),
                dmc.RadarChart(
                    h=300,
                    data=radar_data,
                    dataKey="product",
                    series=[
                        {"name": "sales_january", "color": "lime.4", "opacity": 0.1},
                        {"name": "sales_february", "color": "cyan.4", "opacity": 0.1},
                    ],
                    withPolarGrid=True,
                    withPolarAngleAxis=True,
                    withPolarRadiusAxis=True,
                ),
            ],
            align="center",
        ),
    ],
    forceColorScheme="dark",
)


if __name__ == "__main__":
    app.run_server(debug=True, port=8033)
    ```

**Comment by AnnMarieW** [comment, marked as answer]
Be sure to set the width of the RadarChart.  Note - this applies to all charts - and some other components such as Dash AG Grid
